source/features/toggle-files-button.tsx

```diff
@@ -87,6 +87,9 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isRepoTree,
 	],
+	exclude: [
+		pageDetect.isRepoFile404,
+	],
```

Comment:
Oof these are a lot of manual exclusions. Ok for now but eventually we'd want to silence errors on 404 and empty pages.

- https://github.com/refined-github/refined-github/issues/6558